src/main/scripts/addon

```diff
@@ -99,7 +99,8 @@ if [ -f "$PLF_HOME/addons/addons-manager.jar.new" ]; then
   mv "$PLF_HOME/addons/addons-manager.jar.new" "$PLF_HOME/addons/addons-manager.jar"
 fi
 
-JAVA_OPTS="-Dplf.home=\"$PLF_HOME\""
+JAVA_OPTS=" -Dplf.home=\"$PLF_HOME\""
+EXO_AM_OPTS="$EXO_AM_OPTS -Dplf.home=\"$PLF_HOME\""
```

Comment:
I don't get the need to repeat `-Dplf.home=...` in this variable

We don't need this line as the new variable `EXO_AM_OPTS` value is already inserted in `JAVA_OPTS` in the previous line
```suggestion
```